builder/docker.go

```diff
@@ -47,7 +47,8 @@ func (b *Builder) GetHostAndUpstreamFormats() (string, string, error) {
 	return string(hostFormat), upstreamFormat, nil
 }
 
-func (b *Builder) getDockerImageName(format string) (string, error) {
+// GetDockerImageName gets the existing image name
+func (b *Builder) GetDockerImageName(format string) (string, error) {
```

Comment:
If you're not calling this in `root.go` anymore, it doesn't need to be exported.  Though I guess it's not _hurting_ anything...